### ISSUE TITLE
Support for Contao 4.13

### ISSUE THREAD

**Issue by tomjpeg** [ContaoBlackForest/contao-encore-bundle#7]
Currently, the extension is unfortunately not usable with Contao 4.13. During the installation there are several errors related to MultiColumnWizard and others.

Is support for 4.13 planned in the near future?

**Comment by fritzmg** [comment]
> During the installation there are several errors related to MultiColumnWizard and others.

Are you sure about that? MCW shouldn't be a problem. However, this extension cannot be installed in Contao 4.13 at all because it still requires Symfony 4 dependencies.

**Comment by tomjpeg** [comment]
Yes, exactly believe with MCW it has nothing to do directly. It is just one of the many errors at the moment.

The colleagues at our company also said it will be due to the Symfony 4 dependencies.

**Comment by baumannsven** [comment]
@NewmansHub I am working on the Contao 4.13 release

**Comment by frontendschlampe** [comment]
@baumannsven if you have a dev version, we can help you test it. :-)

**Comment by Xendiadyon** [comment]
@baumannsven Any news?

**Comment by Xendiadyon** [comment]
@baumannsven How is it going? Perhaps someone in this community can help you with the Contao 4.13 release?

**Comment by agonyz** [comment]
Hey @baumannsven, ich habe das Bundle mal testweise in meiner lokalen Contao 4.13 Installation installiert.
Habe dafür die Composer Abhängigkeiten geupgradet und das Polyfill-Bundle entfernt (contao-community-alliance/contao-polyfill-bundle), da dieses nicht mit Contao 4.13 kompatibel war (Hast du an irgendeiner Stelle polyfills aus diesem Bundle verwendet?)

Habe dann noch den output path von 
`'output_path' => '%kernel.project_dir%/web/layout'`
auf
`'output_path' => '%kernel.project_dir%/public/layout'`
aktualisiert und habe es damit in Contao 4.13 zum Laufen bekommen.
Änderungen kannst Du dir hier ansehen: https://github.com/agonyz/contao-encore-bundle/commit/1b77b85913c2c8df9ddeaa3643b9de967f9de52e

Kann ich dich ggf. irgendwie beim Update auf Contao 4.13 unterstützen? Falls ja, melde dich gerne einfach bei mir.

**Comment by baumannsven** [comment]
Hallo

ich habe die erste funktionieren Version erstellt. Installiert werden muss `contaoblackforest/contao-encore-bundle": "^2.0@dev`.
